frontends/election-manager/src/screens/tally_report_screen.tsx

```diff
@@ -255,7 +257,12 @@ export function TallyReportScreen(): JSX.Element {
           {location.pathname === routerPaths.tallyFullReport && (
             <p>
               <Button
-                disabled={!castVoteRecordFiles.wereAdded || isOfficialResults}
+                disabled={
+                  cvrFilesQuery.isLoading ||
+                  !cvrFilesQuery.data ||
```

Comment:
Another place to handle error explicitly

Reply:
bah, thanks - my search regex wasn't broad enough to include these! will take a look.

FWIW, i think most of these cases don't need specific user-facing error handling, since the cvrFiles query failing isn't show-stopping for those flows, but will audit and see if any should be handled differently

Reply:
(also that query failing probably means something that affects the entire app is going on)

Reply:
Ok sounds good